glean-core/src/upload/mod.rs

```diff
@@ -217,7 +227,7 @@ mod test {
     use crate::{tests::new_glean, PENDING_PINGS_DIRECTORY};
 
     const UUID: &str = "40e31919-684f-43b0-a5aa-e15c2d56a674"; // Just a random UUID.
-    const URL: &str = "http://example.com";
+    const URL: &str = "/submit/app_id/ping_name/schema_version/doc_id";
```

Comment:
nit: this is not really an URL anymore, that's more of a path :)

Reply:
Yes... I agree, i thought about changing this, but didn't. I will also change it in PingRequest object.